Shared/Samples/Choose camera controller/ChooseCameraControllerView.swift

```diff
@@ -0,0 +1,161 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import ArcGISToolkit
+import SwiftUI
+
+struct ChooseCameraControllerView: View {
+    /// A scene with imagery basemap style and a tiled elevation source.
+    @State private var scene: ArcGIS.Scene = {
+        // Creates a scene.
+        let scene = Scene(basemapStyle: .arcGISImageryStandard)
+        
+        // Sets the initial viewpoint of the scene.
+        scene.initialViewpoint = Viewpoint(
+            latitude: .nan,
+            longitude: .nan,
+            scale: .nan,
+            camera: Camera(
+                lookingAt: Point(
+                    x: -109.937516,
+                    y: 38.456714,
+                    spatialReference: .wgs84
+                ),
+                distance: 5500,
+                heading: 150,
+                pitch: 20,
+                roll: 0
+            )
+        )
+        
+        // Creates a surface.
+        let surface = Surface()
+        
+        // Creates a tiled elevation source.
+        let worldElevationServiceURL = URL(
+            string: "https://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer"
+        )!
+        let elevationSource = ArcGISTiledElevationSource(url: worldElevationServiceURL)
+        
+        // Adds the elevation source to the surface.
+        surface.addElevationSource(elevationSource)
+        
+        // Sets the surface to the scene's base surface.
+        scene.baseSurface = surface
+        return scene
+    }()
+    
+    /// A graphics overlay containing an airplane graphic.
+    private static let graphicsOverlay: GraphicsOverlay = {
```

Comment:
```suggestion
    @State private var graphicsOverlay: GraphicsOverlay = {
```
To avoid it being recreated unnecessarily.

Reply:
Instead of having a separate graphic as an extension, I'd suggest putting the plane graphic in the overlay directly.

```swift
/// A graphics overlay containing an airplane graphic.
@State private var graphicsOverlay: GraphicsOverlay = {
    let planePosition = Point(x: -109.937516, y: 38.456714, z: 5000, spatialReference: .wgs84)
    let planeSymbol = ModelSceneSymbol(url: URL.bristol, scale: 100)
    let planeGraphic = Graphic(geometry: planePosition, symbol: planeSymbol)
    let graphicsOverlay = GraphicsOverlay(graphics: [planeGraphic])
    graphicsOverlay.sceneProperties.surfacePlacement = .absolute
    return graphicsOverlay
}()
```